### ISSUE TITLE
Uncaught type error bug, upstream in Gatsby

### ISSUE THREAD

**Issue by ehowey** [ehowey/gatsby-theme-catalyst#410]
**Describe the bug**

Related to: https://github.com/gatsbyjs/gatsby/issues/21139

TypeError: document.getElementById(...) is null

```
  170 | if (document.title) {
  171 |   pageName = document.title
  172 | }
> 173 | const pageHeadings = document
  174 |   .getElementById(`gatsby-focus-wrapper`)
  175 |   .getElementsByTagName(`h1`)
  176 | if (pageHeadings && pageHeadings.length) {
```

**Comment by sidharthachatterjee** [comment]
This was an upstream issue due https://github.com/reach/router/commit/0a8af930398570d07d380ef1b6f80833501c8a29

Fixed and published in `gatsby@2.19.12`

**Comment by ehowey** [comment]
Thanks Sid! Got the changes!
